ros/lib/rbac_interface.py

```diff
@@ -118,3 +120,79 @@ def _is_mgmt_url(path):
     Small helper to test if URL is for management API.
     """
     return path.startswith("/mgmt/")
+
+
+def set_host_groups(rbac_response):
+    """
+    We now also have to store the host group information we get from inventory.
+    This comes in the resource definition within the RBAC response:
+
+    {
+      "resourceDefinitions": [
+        {
+          "attributeFilter": {
+            "key": "inventory.groups",
```

Comment:
@upadhyeammit [based on this doc](https://docs.google.com/document/d/1O87MHvKlPEoH_DHwZw4htvwYlkGMiS-LwLQ_qKsZ5J0/edit#heading=h.1z0ggwl751hw) the key should be `group.id` ? or am I missing something here?